<launch>
 	<node pkg="uav_controller" type="uav_controller_server" name="uav_controller_server" output="screen">
 	    <remap from="/vision_pose/pose" to="/maplab_rovio/T_M_I" />
	</node>
</launch>